contracts/cw721-expiration/src/msg.rs

```diff
@@ -15,114 +15,150 @@ pub struct InstantiateMsg {
+    #[returns(cw721::msg::NftInfoResponse<DefaultOptionMetadataExtension>)]
     NftInfo {
         token_id: String,
         /// unset or false will filter out expired nfts, you must set to true to see them
-        include_invalid: Option<bool>,
+        include_expired_nft: Option<bool>,
     },
+
     /// With MetaData Extension.
     /// Returns the result of both `NftInfo` and `OwnerOf` as one query as an optimization
     /// for clients
-    #[returns(cw721::AllNftInfoResponse<Extension>)]
+    #[returns(cw721::msg::AllNftInfoResponse<DefaultOptionMetadataExtension>)]
     AllNftInfo {
         token_id: String,
         /// unset or false will filter out expired approvals, you must set to true to see them
         include_expired: Option<bool>,
         /// unset or false will filter out expired nfts, you must set to true to see them
-        include_invalid: Option<bool>,
+        include_expired_nft: Option<bool>,
     },
 
     /// With Enumerable extension.
     /// Returns all tokens owned by the given address, [] if unset.
-    #[returns(cw721::TokensResponse)]
+    #[returns(cw721::msg::TokensResponse)]
     Tokens {
         owner: String,
         start_after: Option<String>,
         limit: Option<u32>,
         /// unset or false will filter out expired nfts, you must set to true to see them
-        include_invalid: Option<bool>,
+        include_expired_nft: Option<bool>,
     },
+
     /// With Enumerable extension.
     /// Requires pagination. Lists all token_ids controlled by the contract.
-    #[returns(cw721::TokensResponse)]
+    #[returns(cw721::msg::TokensResponse)]
     AllTokens {
         start_after: Option<String>,
         limit: Option<u32>,
         /// unset or false will filter out expired nfts, you must set to true to see them
-        include_invalid: Option<bool>,
+        include_expired_nft: Option<bool>,
     },
 
+    // -------- below is from cw721/src/msg.rs --------
+    /// Return approval of a given operator for all tokens of an owner, error if not set
+    #[returns(cw721::msg::OperatorResponse)]
+    Operator {
+        owner: String,
+        operator: String,
+        include_expired: Option<bool>,
+    },
+    /// List all operators that can access all of the owner's tokens
+    #[returns(cw721::msg::OperatorsResponse)]
+    AllOperators {
+        owner: String,
+        /// unset or false will filter out expired items, you must set to true to see them
+        include_expired: Option<bool>,
+        start_after: Option<String>,
+        limit: Option<u32>,
+    },
+    /// Total number of tokens issued, including all expired NFTs
+    #[returns(cw721::msg::NumTokensResponse)]
+    NumTokens {},
+
+    #[deprecated(since = "0.19.0", note = "Please use GetCollectionInfo instead")]
+    #[returns(cw721::state::CollectionInfo<cw721::state::DefaultOptionCollectionInfoExtension>)]
+    /// Deprecated: use GetCollectionInfo instead! Will be removed in next release!
+    ContractInfo {},
```

Comment:
@JakeHartnell, added docs for all deprecated query msgs and noting they'll be removed in next release.